src/browser/base/zen-components/ZenWorkspaces.mjs

```diff
@@ -538,6 +628,41 @@ var ZenWorkspaces = new (class extends ZenMultiWindowFeature {
_tc"},{"emoji":"🇹🇩","description":"flags, flag: Chad, flag","name":"flag_td"},{"emoji":"🇹🇫","description":"flag: French Southern Territories, flags, flag","name":"flag_tf"},{"emoji":"🇹🇬","description":"flags, flag, flag: Togo","name":"flag_tg"},{"emoji":"🇹🇭","description":"flag: Thailand, flags, flag","name":"flag_th"},{"emoji":"🇹🇯","description":"flag: Tajikistan, flags, flag","name":"flag_tj"},{"emoji":"🇹🇰","description":"flags, flag: Tokelau, flag","name":"flag_tk"},{"emoji":"🇹🇱","description":"flags, flag: Timor-Leste, flag","name":"flag_tl"},{"emoji":"🇹🇲","description":"flags, flag, flag: Turkmenistan","name":"flag_tm"},{"emoji":"🇹🇳","description":"flag: Tunisia, flags, flag","name":"flag_tn"},{"emoji":"🇹🇴","description":"flags, flag, flag: Tonga","name":"flag_to"},{"emoji":"🇹🇷","description":"flags, flag: Turkey, flag","name":"flag_tr"},{"emoji":"🇹🇹","description":"flags, flag, flag: Trinidad & Tobago","name":"flag_tt"},{"emoji":"🇹🇻","description":"flag: Tuvalu, flags, flag","name":"flag_tv"},{"emoji":"🇹🇼","description":"flag: Taiwan, flags, flag","name":"flag_tw"},{"emoji":"🇹🇿","description":"flag: Tanzania, flags, flag","name":"flag_tz"},{"emoji":"🇺🇦","description":"flags, flag, flag: Ukraine","name":"flag_ua"},{"emoji":"🇺🇬","description":"flags, flag: Uganda, flag","name":"flag_ug"},{"emoji":"🇺🇲","description":"flags, flag, flag: U.S. Outlying Islands","name":"flag_um"},{"emoji":"🇺🇸","description":"flags, flag, flag: United States","name":"flag_us"},{"emoji":"🇺🇾","description":"flags, flag, flag: Uruguay","name":"flag_uy"},{"emoji":"🇺🇿","description":"flags, flag, flag: Uzbekistan","name":"flag_uz"},{"emoji":"🇻🇦","description":"flags, flag: Vatican City, flag","name":"flag_va"},{"emoji":"🇻🇨","description":"flag: St. Vincent & Grenadines, flags, flag","name":"flag_vc"},{"emoji":"🇻🇪","description":"flags, flag, flag: Venezuela","name":"flag_ve"},{"emoji":"🇻🇬","description":"flag: British Virgin Islands, flags, flag","name":"flag_vg"},{"emoji":"🇻🇮","description":"flag: U.S. Virgin Islands, flags, flag","name":"flag_vi"},{"emoji":"🇻🇳","description":"flags, flag: Vietnam, flag","name":"flag_vn"},{"emoji":"🇻🇺","description":"flags, flag: Vanuatu, flag","name":"flag_vu"},{"emoji":"🇼🇫","description":"flag: Wallis & Futuna, flags, flag","name":"flag_wf"},{"emoji":"🇼🇸","description":"flag: Samoa, flags, flag","name":"flag_ws"},{"emoji":"🇽🇰","description":"flags, flag, flag: Kosovo","name":"flag_xk"},{"emoji":"🇾🇪","description":"flags, flag, flag: Yemen","name":"flag_ye"},{"emoji":"🇾🇹","description":"flags, flag, flag: Mayotte","name":"flag_yt"},{"emoji":"🇿🇦","description":"flags, flag: South Africa, flag","name":"flag_za"},{"emoji":"🇿🇲","description":"flags, flag: Zambia, flag","name":"flag_zm"},{"emoji":"🇿🇼","description":"flags, flag: Zimbabwe, flag","name":"flag_zw"},{"emoji":"🏳️‍🌈","description":"flag, rainbow flag, pride, gay, flags","name":"rainbow_flag"},{"emoji":"🏳️‍⚧️","description":"trans, pride, flag, flags","name":"transgender_flag"}];
```

Comment:
Please use the emojis inside `ZenEmojis.mjs`

Reply:
![398257097-3dc002ac-b1f3-4d52-a9db-b3d47a965459](https://github.com/user-attachments/assets/5581d1a5-0c50-4fab-bbce-29857f16f56a)